integration-tests/test_display_name_option.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import uuid
+import logging
+from conftest import loop_until
+
+logger = logging.getLogger(__name__)
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_register_with_display_name(insights_client, fetch_from_inventory):
+    """Test insights-client --register --display-name SOME_NEW_HOSTNAME
+
+    The current display name should appear in inventory
+    """
+    unique_hostname = f"test-qa.{uuid.uuid4()}.csi-client-tools.example.com"
+
+    status = insights_client.run("--register", "--display-name", unique_hostname)
+    assert loop_until(lambda: insights_client.is_registered)
+    assert (
+        "Successfully registered host" in status.stdout
+    ), "An application should inform about successfull registration"
```

Comment:
this is already asserted by the basic registration tests, I don't think there is the need to repeat the checks here too